src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -75,6 +80,13 @@ export const DisconnectWalletModal: FC<DisconnectWalletModalProps> = ({
 }) => {
   const { address, connector: activeConnector } = useAccount()
   const { disconnect } = useDisconnect()
+  const chainId = Number(CHAIN_ID)
```

Comment:
I think you'll want to merge main into your branch and use the new hook @vanillaHill created `useActiveNetwork` to get the active chain ID.

Reply:
Fixed - Used Vanillla's implementation